src/ucs/signal/signal.c

```diff
@@ -8,10 +8,35 @@
 #  include "config.h"
 #endif
 
+#ifndef __USE_GNU
+#  define __USE_GNU /* for sighandler_t */
+#endif
```

Comment:
now that it's move to C file, can we define __USE_GNU unconditionally - since no other file is included before it

Reply:
> now that it's move to C file, can we define __USE_GNU unconditionally - since no other file is included before it

ok, removed